odl/space/npy_tensors.py

```diff
@@ -2159,6 +2190,73 @@ def norm(self, x):
             return float(_pnorm_diagweight(x, self.exponent, self.array))
 
 
+class PerAxisWeighting(Weighting):
```

Comment:
Leave out partial implementation from this PR if possible.

Reply:
I thought I do it here already, but no, I'll make a separate PR.